src/api/java/com/minecolonies/api/research/effects/IResearchEffect.java

```diff
@@ -38,11 +38,31 @@
      */
     TranslationTextComponent getDesc();
 
+    /**
+     * Effect subtitle description.
+     *
+     * @return the Subtitle desc.
+     */
+    String getSubtitle();
+
+    /**
+     * Returns the effect's display value
+     *
+     * @return the display value.
+     */
+    Double getDisplay();
```

Comment:
At least as currently implemented, it's the difference between the effect's strength at that level of effect, and the previous level, ie the relative effect.  If the effect strength goes `[0, 0.1, 0.2, 0.5]`, the display values will be `[0, 0.1, 0.1, 0.3]`.  I'd used the more specific relative/absolute names previously, but that was shot down as imprecise because these values are only and should only be used for display purposes, and their internal implementation would have to become more varied to support the data-driven effect PR-to-PR.

I will copy more of the GlobalResearchEffect jdoc which explains this behavior in greater detail to this location, though.